applications/nrf_desktop/doc/dvfs.rst

```diff
@@ -0,0 +1,90 @@
+.. _nrf_desktop_dvfs:
+
+DVFS module
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Dynamic Voltage and Frequency Scaling (DVFS) module is responsible for switching the frequency and voltage according to the application needs.
+It is a way to optimize power consumption by adjusting the frequency and voltage of the core according to the current system state.
+DVFS is critical for balancing performance and power consumption, especially in battery-operated devices.
+
+In the nRF Desktop, the DVFS module enables the system to use the lowest possible frequency and voltage when connected using classic BLE as it is enough to run this mode.
+However, if performing more computation-consuming tasks DVFS module will dynamically change to a higher frequency to increase performance.
+
+Available operating points (in documentation called frequency):
+
+1. DVFS_FREQ_HIGH - high frequency - 320 MHz, 0.8 V
+#. DVFS_FREQ_MEDLOW - medium-low frequency - 128 MHz, 0.6V
+#. DVFS_FREQ_LOW - low frequency - 64 MHz, 0.5V
+
+
+Module events
+*************
+
+.. include:: event_propagation.rst
+    :start-after: table_dvfs_start
+    :end-before: table_dvfs_end
+
+.. note::
+    |nrf_desktop_module_event_note|
+
+Configuration
+*************
+
+The Dynamic Voltage and Frequency Scaling (DVFS) module is enabled by default for nrf54h20 Soc and can be disabled by setting the :option:`CONFIG_DESKTOP_DVFS` Kconfig option to ``n``.
+The module is only available for nrf54h20 SoC.
```

Comment:
consider changing the order of info (IMO it would be better to firstly now if it is even supported on specific platform rather than how to configure it)
```suggestion
The Dynamic Voltage and Frequency Scaling (DVFS) module is available only for the nRF54H20 SoC.
It is enabled by default and can be disabled using the :option:`CONFIG_DESKTOP_DVFS` Kconfig option.
```

Reply:
I moved "The module is only available for nrf54h20 SoC."
before list of available frequencies so will leave first sentence as it is here.